clustered/server/src/main/java/org/ehcache/clustered/server/ServerStoreImpl.java

```diff
@@ -175,4 +178,11 @@ public int getReprobeLength() {
 
     throw new UnsupportedOperationException("Not supported yet.");
   }
+
+  private void checkPayLoadSize(ByteBuffer payLoad) {
+    if (payLoad.remaining() > pageSource.getPool().getSize()) {
+      throw new OversizeMappingException("Payload (" + payLoad.remaining() +
```

Comment:
Why is this exception not thrown to the client? Resilience strategy?

Reply:
yes..default resiliency strategy also evicts in the correct order so did not do an explicit server side eviction, although again I will wait for @chrisdennis to comment.

Reply:
I *think* this is okay.  It's probably not how I would have done it, but I think it's valid.